cmd/state-offline-installer/runInstall.go

```diff
@@ -78,6 +80,19 @@ func runInstall(out output.Outputer, params *Params) error {
 		return errs.Wrap(err, "Unable to create artifactsPath directory")
 	}
 
+	accepted, err := prompts.PromptOfflineTOS(out, prompter)
+	if err != nil {
+		return errs.Wrap(err, "Error with TOS acceptance")
+	}
+	if !accepted {
+		return locale.NewInputError("offline_tos_not_accepted", "License not accepted")
+	}
+
+	// FIX: This is where I would suggest placing asking the EU for the installation directory
+	//     if installToDir == "" {
+	//     installToDir, err = prompter.Input("Choose installation directory", "Where would you like to install the software?", &installToDir, prompt.InputRequired)
```

Comment:
Please record this somewhere appropriate (eg. jira). Code comments are not issue trackers.